### PR TITLE
fix: errors

### PR DESCRIPTION
**Pull request by Cedric921**
- fix fonts ( By using next/font/google

### CODE REVIEW

**Comment by evereq** [comment]
@Cedric921 how is it going with this PR?

File: apps/web/next.config.js [hunk 1]
```diff
@@ -55,6 +55,19 @@ const nextConfig = {
 			'gauzystage.s3.wasabisys.com'
 		]
 	},
+	// module: {
```
Comment (evereq) [inline apps/web/next.config.js:58]: @Cedric921 it's not needed? Why we keep that commented code in such case?

**Comment by evereq** [comment]
@Cedric921 hm, I think this PR is not rebased well as it contains many other changes / commits etc?

**Comment by Cedric921** [comment]
> @Cedric921 hm, I think this PR is not rebased well as it contains many other changes / commits etc?

Now Fixed

**Pull request merged**